CRT.c

```diff
@@ -22,6 +22,9 @@ in the source distribution for its full text.
 #include "ProvideCurses.h"
 #include "XUtils.h"
 
+#include <ncurses.h>
```

Comment:
Because (n)curses support is cursed, there's the `ProvideCurses.h` header available, which internally switches the various places where the `curses`/`ncurses` headers (with and without wide char support) can live. Thus please don't include `ncurses.h` directly: It will break builds. ;-)